source/signature_subpacket/embedded.cpp

```diff
@@ -49,7 +49,7 @@ namespace pgp::signature_subpacket {
      *  @param  other   The object to compare with
      */
     template <signature_subpacket_type subpacket_type, typename contained_t>
-    bool embedded<subpacket_type, contained_t>::operator==(const embedded &other) const noexcept
+    bool embedded<subpacket_type, contained_t>::operator==(const embedded &other) const
```

Comment:
This function should also remain `noexcept`, I feel.

Reply:
In its current form it can't be, since it calls `contained` which may throw.